Content.Server/Paper/PaperSystem.cs

```diff
@@ -59,6 +59,7 @@ private void OnExamined(EntityUid uid, PaperComponent paperComp, ExaminedEvent a
                 return;
 
             if (paperComp.Content != "")
+                args.Message.PushNewline();
                 args.Message.AddMarkup(
```

Comment:
Isn't there like PushMarkup or something that just does this? I don't have access to my IDE atm.

Reply:
Yeah, the ExaminedEvent has a PushMarkup(); I just mimicked what the code below did, which was to modify the FormattedMessage directly. Will update to PushMarkup().